src/Entity/EntityId.php

```diff
@@ -30,6 +55,92 @@ public function getSerialization() {
 	}
 
 	/**
+	 * Returns an array with 3 elements: the foreign repository name as the first element, the local ID as the last
+	 * element and everything that is in between as the second element.
+	 *
+	 * EntityId::joinSerialization can be used to restore the original serialization from the parts returned.
+	 *
+	 * @param string $serialization
+	 * @return string[] Array containing the serialization split into 3 parts.
+	 */
+	public static function splitSerialization( $serialization ) {
```

Comment:
I'm confused - I thought the EntityIdParser would need splitSerialization, which is removed in Jeroen's PR? I'll have to look at it more closely.